spec/factories/collections.rb

```diff
@@ -11,5 +11,9 @@
     after(:build) do |collection, evaluator|
       collection.apply_depositor_metadata(evaluator.user.user_key)
     end
+
+    trait :public do
```

Comment:
Yes, I used the public trait in the setup of spec/features/catalog_search_spec.rb (line 5), to make a collection that has public visibility.  That's here:
https://github.com/projecthydra-labs/curation_concerns/commit/b94388e7ab2afaf0da2b04f496f9965431943c24#diff-8d02e9d055253b29a5729e0e09bc80ecR5